Source/HTTP/httpcall.h

```diff
@@ -18,6 +18,18 @@ struct HeaderCompare
 
 using HttpHeaders = http_internal_map<http_internal_string, http_internal_string, HeaderCompare>;
 
+#if !HC_NOZLIB
+#if HC_PLATFORM == HC_PLATFORM_WIN32 || HC_PLATFORM == HC_PLATFORM_GDK
+enum class HCCompressionAlgorithm : uint32_t
+{
+    None = 0,
+    Gzip,
+    Deflate,
+    Brotli
+};
```

Comment:
Seems like only GZip and None are used here.  Any reason why you didn't just do something like 'bool m_enableGzipCompression'?

Reply:
Internally it doesn't much matter what we do here.  I wouldn't add options that aren't implemented however, so None and GZip are fine.   On the public surface area, in httpClient.h I would do something like:

STDAPI HCHttpCallRequestEnableCompression(
    _In_ HCCallHandle call,
    HCCompressionType type
) noexcept;
#endif

where HCCompressionType is None, Low, Medium, High.

Reply:
Fixed